model/security/patch_role_mapping/structures.smithy

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+structure PatchRoleMappingStructure{
+    content: PatchOperationList
+}
+
+@input
+structure PatchRoleMapping_Input{
+    @required
+    @httpLabel
+    role: String
+    @required
+    @httpPayload
+    content: PatchRoleMappingStructure
```

Comment:
This operation directly takes a list:
```suggestion
@input
structure PatchRoleMapping_Input{
    @required
    @httpLabel
    role: String
    @required
    @httpPayload
    content: PatchOperationList
```

It is expected that this will cause build errors until #127 is merged